crates/bevy_render/src/render_resource/pipeline_cache.rs

```diff
@@ -410,7 +410,7 @@ impl PipelineCache {
             Some((
                 fragment_module,
                 fragment.entry_point.deref(),
-                &fragment.targets,
+                &fragment.targets[..],
```

Comment:
What's the difference here? `&Vec<>` vs `&[]`? In which case, `fragment.targets.as_slice()` feels more idiomatic?